sdk/src/types/block/capabilities.rs

```diff
@@ -173,25 +181,22 @@ impl<Flag: 'static> Packable for Capabilities<Flag> {
     type UnpackVisitor = ();
 
     fn pack<P: packable::packer::Packer>(&self, packer: &mut P) -> Result<(), P::Error> {
-        if !self.is_none() {
-            self.bytes.pack(packer)?;
-        } else {
-            0_u8.pack(packer)?;
-        }
+        self.bytes.pack(packer)?;
```

Comment:
Is that equivalent? I remember asking to switch to that at some point and there was a reason not to

Reply:
It is now, since there's no longer the possibility of having trailing zeroes in the bytes